modules/addons/hooks.php

```diff
@@ -1,22 +1,23 @@
 <?php
 
-if (!defined("WHMCS"))
-	die("This file cannot be accessed directly");
+if (!defined('WHMCS')) {
+	die('This file cannot be accessed directly');
```

Comment:
Not sure why you changed the `""` characters to `''`. From programming languages, `""` means string, and `''` a character. I'd like to have this style adopted for all the code.

Reply:
I know that I can specify strings with single quotation on PHP '', but I'm not sure why you changed every strings. Might I suppose it's a setting from the IDE/editor you're using?